doc/devel/contrib/rust-and-c++.md

```diff
@@ -9,10 +9,9 @@ However, most tasks will only require you to be familiar with one of the two lan
 TaskChampion implements storage and access to "replicas" containing a user's tasks.
 It defines an abstract model for this data, and also provides a simple Rust API for manipulating replicas.
 It also defines a method of synchronizing replicas and provides an implementation of that method in the form of a sync server.
-TaskChampion provides a C interface via the `taskchampion-lib` crate.
+TaskChampion provides a C interface via the `taskchampion-lib` crate, at `src/tc/lib`.
 
 Other applications, besides Taskwarrior, can use TaskChampion to manage tasks.
-Applications written in Rust can use the `taskchampion` crate, while those in other languages may use the `taskchampion-lib` crate.
 Taskwarrior is just one application using the TaskChampion interface.
 
 You can build TaskChampion locally by simply running `cargo build` in the root of this repository.
```

Comment:
Is this still true? Same for the line below referencing the ../../rust subdirectory.

Reply:
Nope, it's not :)